### PR TITLE
Introduce wayland-ivi-extension

### PR DESCRIPTION
**Pull request by iartemenko**

### CODE REVIEW

File: recipes-dom0/dom0-image-weston/files_rcar/meta-xt-images-extra/recipes-graphics/wayland/wayland-ivi-extension_1.10.90.bb [hunk 1]
```diff
@@ -0,0 +1,26 @@
+# Recipe is cloned from meta-ivi, the Yocto layer for In-Vehicle Infotainment.
```
Comment (andr2000) [inline recipes-dom0/dom0-image-weston/files_rcar/meta-xt-images-extra/recipes-graphics/wayland/wayland-ivi-extension_1.10.90.bb]: you shouldn't modify the original recipe with this comment. instead add these lines to the commit message

File: recipes-dom0/dom0-image-weston/files_rcar/meta-xt-images-extra/recipes-graphics/wayland/wayland-ivi-extension_1.10.90.bb [hunk 2]
```diff
@@ -1,12 +1,12 @@
 # Recipe is cloned from meta-ivi, the Yocto layer for In-Vehicle Infotainment.
-# http://git.yoctoproject.org/cgit/cgit.cgi/meta-ivi/
+# http://git.yoctoproject.org/cgit/cgit.cgi/meta-ivi/ (branch: master)
```
Comment (andr2000) [inline recipes-dom0/dom0-image-weston/files_rcar/meta-xt-images-extra/recipes-graphics/wayland/wayland-ivi-extension_1.10.90.bb]: is this change relevant to this patch?

File: recipes-dom0/dom0-image-weston/files_rcar/meta-xt-images-extra/recipes-graphics/wayland/wayland-ivi-extension_1.10.90.bb [hunk 3]
```diff
@@ -1,12 +1,12 @@
...
 
 SUMMARY = "Wayland IVI Extension"
 DESCRIPTION = "GENIVI Layer Management API based on Wayland IVI Extension"
 HOMEPAGE = "http://projects.genivi.org/wayland-ivi-extension"
 BUGTRACKER = "http://bugs.genivi.org/enter_bug.cgi?product=Wayland%20IVI%20Extension"
 LICENSE = "Apache-2.0"
-LIC_FILES_CHKSUM = "file://LICENSE;md5=176cedb32f48dd58f07e0c1c717b3ea4"
+LIC_FILES_CHKSUM = "file://LICENSE;md5=1f1a56bb2dadf5f2be8eb342acf4ed79"
```
Comment (andr2000) [inline recipes-dom0/dom0-image-weston/files_rcar/meta-xt-images-extra/recipes-graphics/wayland/wayland-ivi-extension_1.10.90.bb]: it is suspicious that license's md5 has changed

File: recipes-dom0/dom0-image-weston/files_rcar/meta-xt-images-extra/recipes-graphics/wayland/wayland-ivi-extension_1.10.90.bb (hunk 1 above)
Comment (iartemenko) [reply recipes-dom0/dom0-image-weston/files_rcar/meta-xt-images-extra/recipes-graphics/wayland/wayland-ivi-extension_1.10.90.bb]: Done

**Comment by andr2000** [comment]
lgtm, rb
